pkg/util/testing/defaults.go

```diff
@@ -47,3 +47,13 @@ func MakeDefaultThreeLevelTopology(name string) *kueuealpha.Topology {
 		Levels(DefaultBlockTopologyLevel, DefaultRackTopologyLevel, corev1.LabelHostname).
 		Obj()
 }
+
+// MakeDefaultNamespace creates a default namespace with name.
+func MakeDefaultNamespace(name string) *corev1.Namespace {
+	return MakeNamespace(name).Obj()
+}
+
+// MakeDefaultNamespaceGenerate creates a default namespace with generate name.
+func MakeDefaultNamespaceGenerate(generateName string) *corev1.Namespace {
+	return MakeNamespace("").GenerateName(generateName).Obj()
+}
```

Comment:
I think it would be better to avoid this since this hides actual test data (namespace).
This seems to be a bit programmable.

Reply:
Can you clarify why this hides the actual test data?

It is similar pattern as for other APIs, so I'm not sure why would it be different. The prefix is passed as a parameter, so it is visible to the developer without going into the function. 

Maybe I would adjust the function names, as maybe `MakeDefaultNamespace` suggests this is "default" namespace. WDYT about:
- `MakeNamespace`
- `MakeNamespaceWithPrefix`

Reply:
The problem is `MakeNamespace("")`. This hides to treat an empty namespace "" as a default.
For testing, it should be dumped.

Reply:
I see, I agree with that

Reply:
I guess having two functions eliminates the need for a little awkward optional parameter in golang that is the name. wdyt?

Reply:
> Any reason to prepare this dedicated function?

Just to reduce the amount of code during initialization.

Reply:
For `MakeNamespaceWithGenerateName(name)` sounds good to me. However, `MakeDefaultNamespace(name)` seems not to do almost... isn't it?

Reply:
> > Any reason to prepare this dedicated function?
> 
> Just to reduce the amount of code.

In testing, redundant can be justified.

Reply:
Ok, so
- `MakeNamespaceWithGenerateName(prefix)`, and 
- `MakeNamespace(name)`

Reply:
SGTM, thank you

Reply:
Done